test/unittests/mongoBackend/mongoQueryContextGeo_test.cpp

```diff
@@ -88,21 +88,24 @@ static void prepareDatabase(void)
   BSONObj A = BSON("_id" << BSON("id" << "A" << "type" << "Point") <<
                    "attrNames" << BSON_ARRAY("pos" << "foo") <<
                    "attrs" << BSON(
-                     "pos" << BSON("type" << "location" << "value" << "2, 3") <<
+                     "pos" << BSON("type" << "location" << "value" << "2, 3" << "md" <<
+                              BSON("location" << BSON("type" << "string" << "value" << "WGS84"))) <<
```

Comment:
Looking this modification, I understand the entities currently created in DB using NGSIv1-based location will not work if Orion is update to a version after this change.

We should think a way of avoiding it... otherwise it would need a data migration process, which use to be painfull.

Reply:
Yes, because now Orion handles metadata in a consistent way. A solution should be reintroducing some "If v1" in creation and/or in the rendering logic (as in the case of WSG84 test above), which leads this PR to be not very useful, if the purpose was to close these issues.

Reply:
In any case, for this specific unit test, that modification is needed to match the assertion about metadata vector size, the way in which the geolocation attribute is stored, and then the geoqueries are preserved, as other unit/functional tests are ok.

Reply:
See general comment: https://github.com/telefonicaid/fiware-orion/pull/3346#issuecomment-433969358

Reply:
Looking to the last version of the PR, I understand this changes were finally reverted (as I suggested). Could you confirm?

Reply:
Yes, I confirm.

Reply:
It was fixed in 68020b7